# Disable the ability to clear the console from react-scripts@3.4.4

diff --git a/node_modules/react-dev-utils/clearConsole.js b/node_modules/react-dev-utils/clearConsole.js
index 2099a57..da5d195 100644
--- a/node_modules/react-dev-utils/clearConsole.js
+++ b/node_modules/react-dev-utils/clearConsole.js
@@ -8,9 +8,9 @@
 'use strict';
 
 function clearConsole() {
-  process.stdout.write(
-    process.platform === 'win32' ? '\x1B[2J\x1B[0f' : '\x1B[2J\x1B[3J\x1B[H'
-  );
+  // process.stdout.write(
+  //   process.platform === 'win32' ? '\x1B[2J\x1B[0f' : '\x1B[2J\x1B[3J\x1B[H'
+  // );
 }
 
 module.exports = clearConsole;
